cluster-scope/base/core/namespaces/connaisseur/namespace.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
we would need to add this namespace in here
https://github.com/operate-first/apps/blob/ea8fbfbb1b8b41dd19fbb61a5eac8003b2e753b7/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml#L35
to get this setup properly.